keyboards/herevoland/buff75/readme.md

```diff
@@ -0,0 +1,24 @@
+# Buff75
+
+![Buff75](https://imgur.com/pQfzIZr.jpg)
+An 75%/TenKeyLess sized keyboard with 1 RGB.
+
+* Keyboard Maintainer: [HereVoLand](https://github.com/Vem-596)
+* Hardware Supported: buff75
+* Hardware Availability: [BUFF Customer-Keyboard](https://shop107132374.taobao.com)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make herevoland/buff75:default   
```

Comment:
Previous suggestion here would have been to remove trailing whitespace not the newline
```suggestion
    make herevoland/buff75:default

```